pkg/volume/azure_file/azure_file.go

```diff
@@ -104,6 +105,13 @@ func (plugin *azureFilePlugin) newMounterInternal(spec *volume.Spec, pod *v1.Pod
 	if err != nil {
 		return nil, err
 	}
+	nameSpace := pod.Namespace
+	if spec.PersistentVolume != nil {
+		ns, err := getSecretNamespace(plugin.host.GetKubeClient(), spec.PersistentVolume)
```

Comment:
@liggitt I haven't found that pattern of putting secret namespace in PV yet, the existing [PV annotations](https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/volumehelper/volumehelper.go#L30) don't have one for secret namespace, while we have precedence of storage class name resolution in [mounter](https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/glusterfs/glusterfs.go#L159) and [deleter](https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/storageos/storageos.go#L175)